### ISSUE TITLE
Possible to translate "numbers" to text?

### ISSUE THREAD

**Issue by UltraHQ** [jacopotediosi/GoogleDialerMod#73]
## Summary
In apps like google clock or google photos, instead of boolean text we get random numbers 

## Solution
A way to translate the numbers to text, maybe it's possible?

![Screenshot_20230511-141913](https://github.com/jacopotediosi/GoogleDialerMod/assets/35925402/6682872f-c84a-4573-88dd-0d43ce9d2e52)
![Screenshot_20230511-141922](https://github.com/jacopotediosi/GoogleDialerMod/assets/35925402/d9fd7a37-da54-449e-8db5-44ca2094ab27)

**Comment by jacopotediosi** [comment]
Yeah, they probably are the ID numbers from Google's internal bug ticketing system.

It is planned to add a button for each flag that lets you know what it does in detail, but in the case of apps where the flags have numeric names, it will take a lot of reverse engineering work.

**Comment by m11751** [comment]
> Yeah, they probably are the ID numbers from Google's internal bug ticketing system.是的，它们可能是来自 Google 内部错误通知系统的 ID 号。
> 
> It is planned to add a button for each flag that lets you know what it does in detail, but in the case of apps where the flags have numeric names, it will take a lot of reverse engineering work.计划为每个标志添加一个按钮，让您详细了解它的功能，但对于标志具有数字名称的应用程序，这将需要大量的逆向工程工作。

now I meet this issue too so how to fix this bug now?

**Comment by jacopotediosi** [comment]
> > Yeah, they probably are the ID numbers from Google's internal bug ticketing system.是的，它们可能是来自 Google 内部错误通知系统的 ID 号。
> > It is planned to add a button for each flag that lets you know what it does in detail, but in the case of apps where the flags have numeric names, it will take a lot of reverse engineering work.计划为每个标志添加一个按钮，让您详细了解它的功能，但对于标志具有数字名称的应用程序，这将需要大量的逆向工程工作。
> 
> now I meet this issue too so how to fix this bug now?

This is not a bug. This is expected behavior. Did you read the text you quoted?

**Comment by m11751** [comment]
> > > Yeah, they probably are the ID numbers from Google's internal bug ticketing system.是的，它们可能是来自 Google 内部错误通知系统的 ID 号。
> > > It is planned to add a button for each flag that lets you know what it does in detail, but in the case of apps where the flags have numeric names, it will take a lot of reverse engineering work.计划为每个标志添加一个按钮，让您详细了解它的功能，但对于标志具有数字名称的应用程序，这将需要大量的逆向工程工作。
> > 
> > 
> > now I meet this issue too so how to fix this bug now?
> 
> This is not a bug. This is expected behavior. Did you read the text you quoted?

Sorry,I had read ，my phone is Pixel2xl, so how to fix it ?
